iputils_srcs/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+rm -rf iputils
+
+# echo "Downloading the source code"
+git clone https://github.com/iputils/iputils iputils
+
+# echo "Configuring..."
+cd iputils
+./configure && meson build
+
+# echo "Installing the build..."
+cd builddir && meson install
+
+# echo "Copying ping, tracepath, clockdiff, arping..."
+mkdir -p ../../bin
+cp ./arping ./clockdiff ./tracepath ./ping/ping ../../bin/
+
+cd ../..
```

Comment:
```suggestion
popd &> /dev/null
```
You might need to do `popd` 2 times here.